applications/trilinos_application/custom_strategies/convergencecriterias/trilinos_displacement_criteria.h

```diff
@@ -278,9 +278,11 @@ class TrilinosDisplacementCriteria : public ConvergenceCriteria< TSparseSpace, T
         TDataType local_ReferenceDispNorm = TDataType();
         TDataType temp;
 
+        const double rank = rModelPart.GetCommunicator().MyPID(); // double because I want to compare with PARTITION_INDEX
+
         for(typename DofsArrayType::iterator i_dof = rDofSet.begin() ; i_dof != rDofSet.end() ; ++i_dof)
```

Comment:
I would suggest it_dof (because it is an iterator). and you can make it more compact with auto

Reply:
The code was already there, but OK.